src/main/java/org/folio/rest/impl/InvoiceStorageImpl.java

```diff
@@ -46,7 +62,35 @@ public void getInvoiceStorageInvoices(int offset, int limit, String query, Strin
   @Validate
   @Override
   public void postInvoiceStorageInvoices(String lang, Invoice entity, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-    PgUtil.post(INVOICE_TABLE, entity, okapiHeaders, vertxContext, PostInvoiceStorageInvoicesResponse.class, asyncResultHandler);
+    try {
+      vertxContext.runOnContext(v -> {
+      	log.debug("Creating a new invoice");
```

Comment:
indentation and this is not a big deal, but it's generally good practice to gate logging to debug... e.g.

```
if( will log to debug) {
  log.debug(...);
}
```

Reply:
If I do something like `if (log.isDebugEnabled()) {
      	  log.debug("Creating a new invoice");
      	}` then [sonarcloud code coverage](https://sonarcloud.io/code?id=org.folio%3Amod-invoice-storage&line=67&pullRequest=7&selected=org.folio%3Amod-invoice-storage%3Asrc%2Fmain%2Fjava%2Forg%2Ffolio%2Frest%2Fimpl%2FInvoiceStorageImpl.java)  complains. I can remove least priority log.debug() if that is fine or is there a better alternative?

Reply:
As discussed on Slack, I have changed it to `log.info`

Reply:
Ideally these info messages would be a little more context or detail as that might make them ore useful.  That said, we've spent enough time on this - I'm fine with proceeded as-is.